x-pack/legacy/plugins/epm/public/components/version.tsx

```diff
@@ -6,17 +6,10 @@
 
 import React from 'react';
 import styled from 'styled-components';
-import { RequirementVersion } from '../../common/types';
 
 const CodeText = styled.span`
   font-family: ${props => props.theme.eui.euiCodeFontFamily};
 `;
-export function Version({
-  className,
-  version,
-}: {
-  className?: string;
-  version: RequirementVersion;
-}) {
+export function Version({ className, version }: { className?: string; version?: string }) {
```

Comment:
The [EPR type](https://github.com/elastic/package-registry/pull/196/files#diff-d96ec7ae5bb4a0907779f25ed03acb33R5) & [golang code](https://github.com/elastic/package-registry/blob/master/util/package.go#L53) have `versions` as optional